<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYFeatureMergeDockWidget</class>
 <widget class="QDockWidget" name="QPANSOPYFeatureMergeDockWidget">
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QPANSOPY Feature Merge</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="selectedLayersLabel">
         <property name="text">
          <string>Selected Layers:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="selectedLayersInfo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Select 2+ vector layers in Layers panel</string>
         </property>
         <property name="styleSheet">
          <string>color: #666; font-style: italic;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="parametersGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QFormLayout" name="parametersLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="layerNameLabel">
         <property name="text">
          <string>Merged Layer Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="layerNameLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Merged_Layer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QFormLayout" name="outputLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="outputFolderLabel">
         <property name="text">
          <string>Output Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="outputFolderLayout">
         <item>
          <widget class="QLineEdit" name="outputFolderLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="actionsGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QVBoxLayout" name="actionsLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="buttonsLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QCheckBox" name="exportKmlCheckBox">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>KML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calculateButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Merge Layers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="copyParamsButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Copy Parameters as JSON</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
